scylla/src/statement/query.rs

```diff
@@ -108,6 +108,14 @@ impl Query {
     pub fn get_tracing(&self) -> bool {
         self.config.tracing
     }
+
+    pub fn set_timestamp(&mut self, timestamp: Option<i64>) {
```

Comment:
Please put a docstring similarly to what you did for `PreparedStatement::set_timestamp`.

Reply:
Done